### DISCUSSION TITLE
CSV Local Mode Feature Request

### DISCUSSION THREAD

**Discussion by thanissi** [grafana/grafana-infinity-datasource#859] [Ideas]
Given CSV data source is now in maintenance mode (https://grafana.com/grafana/plugins/marcusolsson-csv-datasource/), request a local mode feature to be made available in Infinity data source - https://grafana.com/docs/plugins/marcusolsson-csv-datasource/latest/configuration/#allow-local-mode

**Comment by Minimalistikaa** [comment]
Would like to have that as well!
This is the only reason why i keep sticking with the csv plugin
